src/main/kotlin/racingcar/Runner.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+import racingcar.domain.RacingGame
+import racingcar.ui.InputView
+
+fun main(args: Array<String>) {
+    val numberOfCars = InputView.getNumberOfCars()
+    val tryCount = InputView.getTryCounts()
+
+    RacingGame(numberOfCars, tryCount).runRace()
```

Comment:
```suggestion
    RacingGame(numberOfCar=numberOfCars, tryCount=tryCount).runRace()
```
numberOfCar과 tryCount의 순서가 바뀐 것 같네요.